kie-server-parent/kie-server-services/kie-server-services-openshift/src/main/resources/META-INF/services/org.kie.server.services.api.StartupStrategy

```diff
@@ -0,0 +1,2 @@
+org.kie.server.services.impl.LocalContainersStartupStrategy
```

Comment:
remove LocalContainerStartupStrategy as it is already defined in kie-server-services-common module so no need to duplicate it here

Reply:
Removed and committed